website/docs/pact_nirvana/step_1.md

```diff
@@ -1,9 +1,17 @@
 ---
-title: 1. Talk
+title: 1. Get prepared - learn about Pact
+sidebar_label: 1. Get prepared - learn about Pact
 ---
 
-Contracts are not a replacement for good communication between or within teams. In fact, contracts require collaboration and communication. One could make the argument that this is one of the main reasons to leverage Pact and enforce communication pathways in large internal and external development organizations.
+Before you read this document, you should:
 
-Contracts are not a magical silver bullet that will allow you to hide in your developer caves and toss built artifacts at each other until everything passes. It is important for all teams to be invested in the process. One of the most common reasons that Pact fails to be successfully adopted in an organisation is a lack of buy in from all parties.
+* have a basic understanding of the concepts of both [consumer driven contracts](https://martinfowler.com/articles/consumerDrivenContracts.html) and Pact,
+* have read the [Pact Broker Overview](pact_broker/overview.md)
+* have read the section on [versioning in the Pact Broker](getting_started/versioning_in_the_pact_broker.md)
+* Do the CI/CD workshop
 
-Collaborate about the problems, collaborate over the design, and keep the communication channels open.
+Before setting up Pact in your own environment, it can help to have an idea of what a working pipeline might look like. 
+
+Completing the [Pact CI/CD workshop](https://docs.pactflow.io/docs/workshops/ci-cd/) will give you a good understanding of how the Pact client libaries interact with the Pact Broker and where these interactions fit into your release pipeline.
+
+> Note that if you are using your own instance of the open source Pact Broker, it does not support secrets, and it does not have a user interface for managing webhooks. You'll need to use the API to create the webhook, and your Travis CI token will have to be stored in plain text in the webhook.
```

Comment:
Not quite true. You can use the HAL browser to create webhooks...

Reply:
This came from Beth's original cut [here](https://github.com/pact-foundation/docs.pact.io/blame/37c0e9d09ad0cfdde87767fea5fa25e5b8be832d/docs/pact_nirvana/step_4.md#L29). not sure if we have details in the docs as to how to do it or may just be out of date info

Reply:
I don't think it's documented, but once you learn how to use the HAL browser it goes pretty quickly.

Also, I see you mention Travis CI but for example we're using Jenkins, so do we want to mention specific CI tools?

Reply:
personally, not really on the specific CI tools (too many and we should be tool agnostic in this guide I think). I think I will remove the travis mention but retain the message that the CI token will be in plain text, and suggest that the user can use the API or HAL Browser to add webhooks 👍🏾

Reply:
Updated copy

> Note that if you are using your own instance of the open source Pact Broker, it does not support secrets, and it does not have a user interface for managing [webhooks](https://docs.pact.io/pact_broker/webhooks). You'll need to use the API or HAL Browser to create the webhook, and your CI token will have to be stored in plain text in the webhook. See the Webhooks API reference docs [here](https://docs.pact.io/pact_broker/advanced_topics/api_docs/webhooks).